Buildings/Fluid/Air/BaseClasses/PartialAirHandlingUnit.mo

```diff
@@ -0,0 +1,231 @@
+within Buildings.Fluid.Air.BaseClasses;
+partial model PartialAirHandlingUnit "Partial AHU model "
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+    final show_T=true);
+  extends Buildings.Fluid.Air.BaseClasses.EssentialParameter;
+  extends Buildings.Fluid.Actuators.BaseClasses.ValveParameters(
+    final m_flow_nominal=m1_flow_nominal,
+    final rhoStd=Medium1.density_pTX(101325, 273.15+4, Medium1.X_default),
+    final deltaM = deltaM2);
+  extends Buildings.Fluid.Interfaces.FourPortFlowResistanceParameters(
+    final computeFlowResistance1=true,
+    final computeFlowResistance2=true);
+
+  // Cooling coil
+  parameter Boolean waterSideFlowDependent=true
```

Comment:
follow the Parameter naming conventions, i.e. a combination of the first three characters of a word. so the name `waterSideFlowDependent` should change to `watSidFloDep`.